app/models/rate_start.rb

```diff
@@ -2,7 +2,7 @@
 
 class RateStart < ApplicationRecord
 
-  has_many :duration_rates
+  has_many :duration_rates, dependent: :destroy
```

Comment:
This change doesn't actually belong here, but it's needed. I can undo it and add this in the next PR

Reply:
Smaller PRs are good - if this isn't related to model changes, save it for later